cmip6_downscaling/config/config.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+os.environ['PREFECT__FLOWS__CHECKPOINTING'] = 'true'
+from dask_kubernetes import KubeCluster, make_pod_spec
+from funnel import CacheStore
+from prefect.executors import DaskExecutor
+from prefect.run_configs import KubernetesRun
+from prefect.storage import Azure
+
+# Azure --------------------------------------------------------------------
+connection_string = os.environ.get("AZURE_STORAGE_CONNECTION_STRING")
+storage = Azure("prefect")
+cache_store = CacheStore('az://flow-outputs/bcsd')
+serializer = 'xarray.zarr'
+
+# Prefect --------------------------------------------------------------------
+# prefect agent ID
+agent = ["az-eu-west"]
+
+# Docker image
+image = "carbonplan/cmip6-downscaling-prefect:2021.12.06"
+
+# Kubernetes config
+kubernetes_cpu = 7
+kubernetes_memory = "16Gi"
+
+# Dask executor config
+dask_executor_memory_limit = "16Gi"
+dask_executor_memory_request = "16Gi"
+dask_executor_threads_per_worker = 2
+dask_executor_cpu_limit = 2
+dask_executor_cpu_request = 2
+dask_executor_adapt_min = 4
+dask_executor_adapt_max = 20
+
+extra_pip_packages = {
+    "EXTRA_PIP_PACKAGES": "git+https://github.com/carbonplan/cmip6-downscaling.git@param_json git+https://github.com/orianac/scikit-downscale.git@bcsd-workflow"
```

Comment:
how about we install the main branch of scikit-downscale and change the BCSD model names in this repo to conform